Basic FE project template/ui-src/styles/general/variables/_variables.scss

```diff
@@ -12,3 +12,4 @@ $background-color: #f4f4f4;
 
 // Rating variables
 $color--rating: #202448;
+$color--rating-1: #73758a;
```

Comment:
change the naming if this variable, if I work for the first time on this project, I cannot understand what this name reffers to, or how it's different from $color--rating. You could use bem here, the first variable can be $color-rating, and the second one $color-rating--WHATEVER_THIS_SHOULD_BE

Reply:
Or Why did you call it "rating"? you don't rate anything in the payment section

Reply:
If you look in the BeLeasing template, color: #73758A; is the color for h6, so it should be a general style.